azure-pipelines-cloud-db-integration.yml

```diff
@@ -0,0 +1,112 @@
+stages:
+  - stage: lint
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    jobs:
+      - job: lint
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: 3.7
+            displayName: 'Use Python 3.7'
+
+          - script: |
+              pip install isort[requirements]==5.4.2 flake8==3.8.3 black==20.8b1 pyupgrade==2.7.2
+              EXIT_STATUS=0
+              isort . --check-only --skip docs/ || EXIT_STATUS=$?
+              black --check --exclude docs/ . || EXIT_STATUS=$?
+              flake8 great_expectations/core || EXIT_STATUS=$?
+              pyupgrade --py3-plus || EXIT_STATUS=$?
+              exit $EXIT_STATUS
+
+  - stage: cloud_db_integration_expectations_cfe
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    dependsOn: [lint]
```

Comment:
On second thought, do we actually require `lint`? We are doing this check on `develop`, a branch that strictly disallows code to be merged if `lint` fails on its own pipeline. As such, aren't we guaranteed to have properly linted code by the time this check rolls around?

Reply:
Great point. Linting is **definitely** handled by this point